pkg/transforms/vapbinding.go

```diff
@@ -0,0 +1,60 @@
+/*
+IBM Confidential
+OCO Source Materials
+(C) Copyright IBM Corporation 2019 All Rights Reserved
+The source code for this program is not published or otherwise divested of its trade secrets,
+irrespective of what has been deposited with the U.S. Copyright Office.
+Copyright (c) 2020 Red Hat, Inc.
+*/
+// Copyright Contributors to the Open Cluster Management project
+
+package transforms
+
+import (
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+type VapBindingResource struct {
+	node Node
+}
+
+func VapBindingResourceBuilder(v *unstructured.Unstructured) *VapBindingResource {
+	node := transformCommon(v)         // Start off with the common properties
+
+	typeMeta := metav1.TypeMeta{
+		Kind:       v.GetKind(),
+		APIVersion: v.GetAPIVersion(),
+	}
+	
+	apiGroupVersion(typeMeta, &node) // add kind, apigroup and version
+	// Extract the properties specific to this type
+
+	node.Properties["validationActions"], _, _ = unstructured.NestedStringSlice(v.Object, "spec", "validationActions")
+	node.Properties["policyName"], _, _ = unstructured.NestedString(v.Object, "spec", "policyName")
+
+	owners := v.GetOwnerReferences()
+	fromGK := false
+
+	for _, o := range owners {
+		if strings.HasPrefix(o.APIVersion, "constraints.gatekeeper.sh") {
+			fromGK = true
+		}
+	}
+
+	node.Properties["fromGatekeeper"] = fromGK
```

Comment:
Do we expect users to search by this property? If no, then change it to `_fromGatekeeper`.

Reply:
Would `_ownedByGatekeeper` be more descriptive?

Reply:
great